_automating-configurations/api/delete-workflow.md

```diff
@@ -7,9 +7,11 @@ nav_order: 80
 
 # Delete a workflow
 
-When you no longer need a workflow template, you can delete it by calling the Delete Workflow API. 
+When you no longer need a workflow template, you can delete it by calling the Delete Workflow API.
 
-Note that deleting a workflow only deletes the stored template but does not deprovision its resources.  
+Note that deleting a workflow only deletes the stored template but does not deprovision its resources.
```

Comment:
```suggestion
Note that deleting a workflow only deletes the stored template---it does not deprovision its resources.
```

Reply:
Fixed... confirming three (3) hyphens is intended here?

Reply:
3 hyphens is a Markdown way to specify an em dash 😄